Description: Respect '--disable-schemas-install'
 Make configure flags '--disable-schemas-install' be respected during
 'make install'.
Author: Nicolas Bourdaud <nicolas.bourdaud@gmail.com>
Origin: vendor
Forwarded: no
Last-Update: 2012-07-06

--- a/data/Makefile.am
+++ b/data/Makefile.am
@@ -77,8 +77,10 @@
 shaders_DATA = \
 	shaders/dim-window.glsl
 
+if GCONF_SCHEMAS_INSTALL
 install-data-local:
 	GCONF_CONFIG_SOURCE=$(GCONF_SCHEMA_CONFIG_SOURCE) $(GCONFTOOL) --makefile-install-rule $(srcdir)/$(gconfschema_DATA)
+endif
 
 
 
